spring/src/test/kotlin/de/codecentric/hikaku/converters/spring/deprecation/ConsumesTestController.kt

```diff
@@ -0,0 +1,33 @@
+package de.codecentric.hikaku.converters.spring.deprecation
```

Comment:
File is still named `ConsumesTestController.kt` please rename it to `NoDeprecationController.kt`